kratos/python_scripts/from_json_check_result_process.py

```diff
@@ -134,14 +137,14 @@ def ExecuteFinalizeSolutionStep(self):
             for node in self.sub_model_part.Nodes:
                 compute = self.__check_flag(node)
 
-                if (compute == True):
+                if (compute is True):
```

Comment:
I am curious, does this make a difference?

Reply:
Is more pythonic

Reply:
https://blog.startifact.com/posts/older/what-is-pythonic.html

Reply:
`is` checks that both sides point to the same object, `==` only checks values. It does make a difference. I would not go for `is` here, this would fail if `compute` is set to a value that normally evaluates to `True` (such as `1`).  It also relies on the somewhat obscure fact that boolean values are singletons in python. If you really want to feel pythonic write `if compute:` :P .

https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is-in-python

Reply:
I only know that if I activate the PEP8 check in Kdevelop it recommends this.

Trivia each time I read PEP8 I think in this: https://www.youtube.com/watch?v=m0lpgFKVrgQ

Reply:
This is actually from PEP8:
>- Don't compare boolean values to True or False using ==.
>Yes:   if greeting:
>No:    if greeting == True:
>Worse: if greeting is True:

So I don't know why Kdevelop would say that...

As always, SO has more info on why https://stackoverflow.com/questions/4050335/strange-pep8-recommandation-on-comparing-boolean-values-to-true-or-false